gem2rpm/foreman_plugin.spec.erb

```diff
@@ -0,0 +1,184 @@
+# FIXME:
+#   1. Change scl_prefix to _ruby or _ror for Requires/BuildRequires that are
```

Comment:
Nice, I suppose we can go the extra mile after this PR & fetch the ror/ruby packages list so that these are also set automatically. :smile: